concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-586	586	terroso	earthy	1.412	2.235	4	0.618	4.676	4.676	vista	12.941	1.421	0.314	3.412	4.235	3.444		
+Morucci-2019-643-587	587	tiepido	lukewarm	2.941	0.676	4.5	0.324	1.206	4.5	tatto	9.647	1.278	0.433	6.333	3.1	6.316		
+Morucci-2019-643-588	588	tintinnante	tinkling	0.05	0.05	1.075	4.821	2.175	4.821	udito	8.171	0.993	0.584	4.412	4.938	4.778		
+Morucci-2019-643-589	589	tinto	stained	0.135	0.568	0.351	0	4.865	4.865	vista	5.919	0.64	0.822	5.765	4.471	5.667		
+Morucci-2019-643-590	590	tipico	typical	2.375	2.15	1.8	2.3	3.15	3.15	vista	11.775	1.592	0.115	6.684	3.824	5.316		
+Morucci-2019-643-591	591	tonante	thunderbolt	0	0.171	0.314	4.857	1.743	4.857	udito	7.086	0.832	0.685	2.778	5.85	3.053	3760	THUNDERY
+Morucci-2019-643-592	592	tondo	round	0.378	0.108	4.486	0.432	4.892	4.892	vista	10.297	1.018	0.465	6.579	2.765	6.389		
+Morucci-2019-643-593	593	torbato	peat	2.452	2.613	0.613	0.29	2.71	2.71	vista	8.677	1.383	0.279	2.235	6.364	2.778		
+Morucci-2019-643-594	594	torbido	cloudy	0.744	1.051	1.103	0.526	4.59	4.59	vista	8.013	1.258	0.507	5.647	5.389	4.944	2341	CLOUDY
+Morucci-2019-643-595	595	tostato	toasted	4.412	4.147	2.382	0.647	4.059	4.412	gusto	15.647	1.477	0.241	5.667	4.7	5.895		
+Morucci-2019-643-596	596	tozzo	stumpy	0.132	0.051	3.949	0.256	4.692	4.692	vista	9.08	0.895	0.511	4.5	5	5		
+Morucci-2019-643-597	597	trascendente	transcendent	0.231	0.308	0.436	1.667	1.564	1.667	udito	4.205	1.32	0.341	4.389	6.75	3.105		
+Morucci-2019-643-598	598	traslucido	translucent	0	0.075	1.125	0.225	4.925	4.925	vista	6.35	0.674	0.776	2.611	6.158	2.632		
+Morucci-2019-643-599	599	trasparente	transparent	0	0.027	0.622	0.405	4.973	4.973	vista	6.027	0.599	0.825	6.235	3.4	6.053	3754	TRANSPARENT
+Morucci-2019-643-600	600	tremendo	awful	2.675	2.675	1.15	3.475	3.55	3.55	vista	13.525	1.551	0.177	6.706	3.167	5.722		
+Morucci-2019-643-601	601	tremolante	flickering	0.205	0.077	3.744	2.667	4.641	4.641	vista	11.333	1.178	0.403	5.647	4.5	5.059		
+Morucci-2019-643-602	602	triangolare	triangular	0.128	0.051	4.359	0.128	4.872	4.872	vista	9.538	0.845	0.505	6.294	3.667	5.833		
+Morucci-2019-643-603	603	trilaterale	trilateral	0.2	0.05	3.875	0.475	4.75	4.75	vista	9.35	0.971	0.503	2.833	6.05	3.105		
+Morucci-2019-643-604	604	turchese	turquoise	0.103	0.128	0.054	0.128	4.974	4.974	vista	5.387	0.373	0.913	4.278	3.75	5.526		
+Morucci-2019-643-605	605	turchino	turquoise	0	0.028	0.028	0.056	4.972	4.972	vista	5.083	0.128	0.978	3.053	4.188	4.211		
+Morucci-2019-643-606	606	udibile	audible	0.028	0.028	0.389	4.972	1.139	4.972	udito	6.556	0.728	0.754	3.833	4.95	5.211		
+Morucci-2019-643-607	607	ululante	howling	0	0	0.189	5	2.541	5	udito	7.73	0.738	0.647	4.353	4.444	4.944		
+Morucci-2019-643-608	608	umidiccio	moist	0.95	1.725	4.725	0.625	3.725	4.725	tatto	11.75	1.372	0.349	4.474	4.294	5.053	143	HUMID
+Morucci-2019-643-609	609	umido	moist	1.553	2.289	4.316	0.263	3.514	4.316	tatto	11.935	1.394	0.34	6.824	3.944	6.556		
```

Comment:
This is the correct HUMID; "umidiccio" is a restrictive ("sort of humid", "a little humid")